_assignments/mp2.md

```diff
@@ -0,0 +1,186 @@
+---
+layout: assignment
+title: "MP2: Geometric Transformations / Perception"
+parts:
+  -
+    title: MP2.1 (to become MP3)
+    due: Mar. 14th, 2019 @ 11:59 PM
+  -
+    title: MP2.2 (to become MP4)
+    due: Mar. 21st, 2019 @ 11:59 PM
+material:
+  -
+    name: Starting Unity package
+    url: https://courses.engr.illinois.edu/cs498vr3/fa2018/mps/MP1.zip
+rubric:
+  -
+    name: View Flipping
+    points: 10
+    description: Flipping the camera works properly and preserves the user's global position.
+  -
+    name: Position Reset
+    points: 10
+    description: The position reset functionality works properly.
+  -
+    name: Cube Match Mode
+    points: 20
+    description: The cube correctly matches the user's movements.
+  -
+    name: Cube Mirror Mode
+    points: 20
+    description: The cube correctly mirrors the user's movements.
+  -
+    name: Cube Mode Toggle
+    points: 10
+    description: The cube switches between mirror and match modes correctly.
+  -
+    name: Geometric Transformations
+    points: 40
+    description: The answers to the written problems are correct.
+  -
+    name: Position Tracking
+    points: 10
+    description: Toggling position tracking works correctly, and does not cause discontinuities.
+  -
+    name: Rotation Tracking
+    points: 40
+    description: Toggling rotation tracking works correctly, regardless of where we are looking.
+  -
+    name: Depth Perception Trick
+    points: 20
+    description: |
+      The spheres appear to be of equal size while the script is active,
+      regardless of where the user is in the room.
+  -
+    name: Depth Perception Script
+    points: 20
+    description: The spheres appear in the correct order, and the script works when run repeatedly.
+points: 200
+---
+
+MP2.1 consists of [VR Mirror](#vr-mirror) and [Geometric Transformations](#geometric-transformations). This section will be unnecessary once we approve splitting these MPs up.
+
+<hr />
```

Comment:
forgot to mention this earlier, but you can use three dashes `---` to insert a line, keeping in Markdown